setup.cfg

```diff
@@ -122,6 +122,8 @@ install_requires =
     python-dateutil>=2.3, <3
     python-nvd3~=0.15.0
     python-slugify>=3.0.0,<5.0
+    # Required for flask-openID which comes with flask-appbuilder
```

Comment:
Can you add a comment here on which we have specifically added it (about poetry) and maybe a link to this PR or one of your comments

Reply:
Sure